tests_e2e/GuestAgentDcrTestExtension/Utils/test/MockUtil.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
```

Comment:
are the unit test running?

Reply:
no they're not, added todo comment to the code that these are currently for reference